user/enterprise/platform-tips.md

```diff
@@ -393,6 +391,32 @@ $ curl -H "Authorization: token XXXXX" https://travis.example.com/api/uptime
 If everything is up and running, it answers with a `HTTP 200 OK`, or in case of failure with a `HTTP 500 Internal Server Error`.
 
 
+## Configuring Backups
```

Comment:
I mentioned this in the commit message but I feel that this fits better in this file than the Operations Manual/Troubleshooting Guide. This section isn't an 'issue', really, and the rest of the troubleshooting guide contains instructions on how to fix issues.

Reply:
:+1:

Reply:
👍